### DISCUSSION TITLE
Recon before setup manager starts

### DISCUSSION THREAD

**Discussion by jeden-92** [jamf/Setup-Manager#66] [Q&A]
I have a query around the order of operations with setup manager, if I scope policies or profiles to specific macOS versions via a smart group is setup manager smart enough to run an inventory before it starts so that smart groups are correctly populated? Otherwise if no inventory is submitted a device won't be in the correct smart group and therefor won't be inscape of specific policies.

Let me know if this actually already part of the Jamf binary, I have had mixed results with Jamf onboarding where policies don't install because a smart group is not populated correctly until after the machine submits inventory.

**Comment by scriptingosx** [comment]
[FAQ -> What is happening during "Getting Ready" and "Finishing"](https://github.com/Jamf-Concepts/Setup-Manager/blob/main/Docs/FAQ.md#what-is-happening-during-getting-ready-and-finishing)
